packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -1,9 +1,60 @@
-import { Type, restBaseUrl, validators } from '@openmrs/esm-framework';
+import { Type, validators } from '@openmrs/esm-framework';
 import vitalsConfigSchema, { type VitalsConfigObject } from './current-visit/visit-details/vitals-config-schema';
 import biometricsConfigSchema, {
   type BiometricsConfigObject,
 } from './current-visit/visit-details/biometrics-config-schema';
 
+export const defaultTablesConfig: TablesConfig = {
+  columnDefinitions: [
+    {
+      id: 'patient-name',
+      columnType: 'patient-name-column',
+    },
+    {
+      id: 'patient-id',
+      columnType: 'patient-id-column',
+      config: {
+        identifierType: 'openmrs-id-uuid',
+      },
+    },
+    {
+      id: 'priority',
+      columnType: 'priority-column',
+      config: [],
+    },
+    {
+      id: 'comingFrom',
+      columnType: 'queue-coming-from-column',
+    },
+    {
+      id: 'status',
+      columnType: 'status-column',
+      config: [],
+    },
+    {
+      id: 'queue',
+      columnType: 'current-queue-column',
+    },
+    {
+      id: 'wait-time',
+      columnType: 'wait-time-column',
+    },
+    {
+      id: 'actions',
+      columnType: 'active-visits-actions-column',
+    },
+  ],
+  tableDefinitions: [
+    {
+      columns: ['patient-name', 'priority', 'comingFrom', 'status', 'wait-time', 'actions'],
+      appliedTo: { queue: '3113f164-68f0-11ee-ab8d-0242ac120002' },
```

Comment:
2 different `TableDefinition`s here. The first one is for illustration purpose, to show that we can define the columns for a specific queue. The second one applies to all tables.